storage/src/testFixtures/java/tech/pegasys/teku/storage/server/DatabaseContext.java

```diff
@@ -72,6 +74,7 @@ public StorageSystem createFileBasedStorage(
         .version(getDatabaseVersion())
         .storageMode(storageMode)
         .stateStorageFrequency(1L)
+        .asyncRunner(Optional.of(new StubAsyncRunner()))
```

Comment:
this is sufficient for current, but not sure about once we're migrating finalized and actually using the async migration.